src/stores/RoomListStore.tsx

```diff
@@ -28,14 +28,17 @@ export class RoomListStore {
   private readonly _roomService: RoomService = new RoomService();
```

Comment:
아마 제가 이전에 작성한 코드를 따라서 작성하다보니 예전에 이렇게 작성한것 같은데여, 앞서 보인 리뷰를 반영한다면 아래처럼 수정하는게 맞을듯하네요.
```suggestion
  private readonly _roomService: RoomService = roomService;
```

추가적인 설명은 디코로 하겠습니다.